From 4429847e17485fdd3425c7804c2ecadba0f9f8e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Mon, 6 Feb 2023 19:37:30 +0100
Subject: [PATCH] Fix build with gcc 10
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 src/parseopt.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/parseopt.c b/src/parseopt.c
index a7b504f..1f6b561 100644
--- a/src/parseopt.c
+++ b/src/parseopt.c
@@ -255,8 +255,6 @@ print_option_descr (const char *descr, size_t lmargin, size_t rmargin)
 }
 
 char *parseopt_program_name;
-char *parseopt_program_doc;
-char *parseopt_program_args;
 const char *program_bug_address = "<" PACKAGE_BUGREPORT ">";
 void (*parseopt_help_hook) (FILE *stream);
 
-- 
2.39.1

